test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,44 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-apibinding]", func() {
```

Comment:
There is no "sig" called "apibinding". "Sig" is usually used in kubernetes / openshift, seems not seen in kcp. From https://github.com/kcp-dev/kcp/issues , format "area/xxx" is used, like "area/apiexports". I'm not saying you need use "area/xxx", but I'm sure "sig-xxx" seems not proper in kcp.

Reply:
Agree with xxia here, also i am not sure if the package name needs to be apibinding, rather can we call it compute ? If could either be sharedcompute or BYO. WDYT ?

Reply:
Good idea! Fixed.

Reply:
@Phaow do not see anything being fixed here, did you intended to use area/xxx instead of [sig-apibinding] ?

Reply:
@kasturinarra Yeah, I'll accept xingxing's idea.